com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/steps/SyncActivitiStepTest.java

```diff
@@ -79,6 +89,24 @@ public void initMocks() {
         when(clientProvider.getControllerClient(anyString(), anyString(), anyString(), anyString())).thenReturn(client);
         context.setVariable("correlationId", getCorrelationId());
         prepareExecution();
+        prepareProcessEngineConfiguration();
+    }
+
+    private void prepareProcessEngineConfiguration() {

```

Comment:
Can we please split this method to several smaller ones. I know it's a test, but still... :D

Reply:
The tests need to be in perfect shape!!!